__tests__/app.test.js

```diff
@@ -209,4 +209,47 @@ describe("GET: /api/users", () => {
                 expect(body.msg).toBe("Invalid input, no such end point exists");
             })
     })  
+})
+
+describe("GET: /api/articles", () => {
+    it("Status 200: responds with an array of article objects, each of which have the following properties: author, title, article_id, topic, created_at, votes, comment_count ", () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(( {body} ) => {
+                const { articles } = body
+                expect(articles).toBeInstanceOf(Array);
+                expect(articles.length).toBe(12)
+                
+               articles.forEach((article) => {
+                    expect(article).toEqual(
+                        expect.objectContaining({
+                            author: expect.any(String),
+                           title: expect.any(String),
+                           article_id: expect.any(Number),
+                          topic: expect.any(String),
+                          created_at: expect.any(String),
+                          votes: expect.any(Number),
+                          comment_count: expect.any(String)
```

Comment:
comment count should be an integer not string

Reply:
I have updated all tests to account for type number with all tests passing now :)